doc/whats_new.rst

```diff
@@ -133,6 +133,11 @@ Bug fixes
      functions were not accepting multi-label targets. :issue:`7676`
      by `Mohammed Affan`_
 
+   - Fixes issue in :func:`multiclass.OneVsRestClassifier.partial_fit` of
```

Comment:
Take a look at how this renders: https://6948-843222-gh.circle-artifacts.com/0/home/ubuntu/scikit-learn/doc/_build/html/stable/whats_new.html#id3. I think one reference is enough, rather than three!